torch/_inductor/cpu_vec_isa.py

```diff
@@ -163,19 +163,55 @@ def __str__(self) -> str:
 
 
 @dataclasses.dataclass
-class VecSVE(VecISA):
-    # this function can be repurposed for SVE with variable vec length
+class VecSVE128(VecISA):
+    # SVE with 128-bit width
+    _bit_width = 128
+    _macro = [
+        "CPU_CAPABILITY_SVE",
+        "CPU_CAPABILITY_SVE128",
+        "AT_BUILD_ARM_VECSVE_WITH_SLEEF",
+    ]
+    _arch_flags = "-march=armv8.2-a+sve -msve-vector-bits=128"
+    _dtype_nelements = {torch.float: 4, torch.bfloat16: 8, torch.float16: 8}
+
+    def __str__(self) -> str:
+        return "sve128"
+
+    __hash__: Callable[[VecISA], Any] = VecISA.__hash__
+
+
+@dataclasses.dataclass
+class VecSVE256(VecISA):
+    # SVE with 256-bit width
     _bit_width = 256
     _macro = [
         "CPU_CAPABILITY_SVE",
         "CPU_CAPABILITY_SVE256",
-        "AT_BUILD_ARM_VEC256_WITH_SLEEF",
+        "AT_BUILD_ARM_VECSVE_WITH_SLEEF",
```

Comment:
I guess this is not in this diff but wondering why we linked these two build flags like this, as opposed to a dependent CMake option.

Reply:
This macro is defined in the main CMakeLists.txt (https://github.com/pytorch/pytorch/blob/main/CMakeLists.txt#L916) and also I believe this macro flag is required to enable sleef path with torch inductor and had to be defined here as well.
It already existed before for ARM Neon flow in the vec256 Aten backend,
https://github.com/pytorch/pytorch/blob/main/torch/_inductor/cpu_vec_isa.py#L175

We used it as reference and extended for SVE backend flow on ARM.